sda-download/.github/integration/tests/common/70_check_download.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+if [ "$STORAGETYPE" = s3notls ]; then
+    exit 0
+fi
+
+cd dev_utils || exit 1
+
+# get a token, set up variables
+token=$(curl --cacert certs/ca.pem "https://localhost:8000/tokens" | jq -r  '.[0]')
+dataset="https://doi.example/ty009.sfrrss/600.45asasga"
+file="dummy_data"
+expected_size=1048605
+C4GH_PASSPHRASE=$(grep -F passphrase config.yaml | sed -e 's/.* //' -e 's/"//g')
+export C4GH_PASSPHRASE
+
+# download decrypted full file,  check file size
+curl --cacert certs/ca.pem -H "Authorization: Bearer $token" "https://localhost:8443/s3/$dataset/$file" --output full1.bam
+file_size=$(stat -c %s full1.bam)  # Get the size of the file
+
+if [ "$file_size" -ne "$expected_size" ]; then
+    echo "Incorrect file size for full decrypted file"
+    exit 1
+fi
+
+# test that start, end=0 returns the whole file
+curl --cacert certs/ca.pem -H "Authorization: Bearer $token" "https://localhost:8443/s3/$dataset/$file?startCoordinate=0&endCoordinate=0" --output full2.bam
```

Comment:
As I tested, when start=0 and end=0, the command did not return a full file, instead it was terminated without returning any data. Did I make a mistake or is that because the `startCoordinate` has not been implemented?

Reply:
Fixed in pair session